CHANGELOG.md

```diff
@@ -13,17 +13,30 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Breaking changes
 
-- New anchoring implementation based on native segwit transactions. (#136)
+- New anchoring implementation is based on native segwit transactions. Acceptance of an
+  anchoring transaction still requires the majority of votes of the validators -
+  `2/3n+1`. As the votes are now separated from the transaction body, they do not
+  affect transaction hash and, correspondingly, its ID.(#136)
 
   In this way:
 
-  - It is fully determenistic and does not contain LECT messages.
-
-  - It does not need an observer anymore.
-
-  - Exonum nodes can work without connection with the `bitcoind`.
-
-  For more details you can visit the [readme](README.md) page.
+  - Anchoring transactions have become fully deterministic. The problem of transaction
+    [malleability]( https://en.wikipedia.org/wiki/Malleability_(cryptography)) has,
+    thus, been solved. The validators do not have to agree on the latest Exonum
+    transaction anchored to Bitcoin blockchain, which we called LECT, any more to
+    continue the anchoring chain.
```

Comment:
> The validators do not have to agree on the latest Exonum
    transaction anchored to Bitcoin blockchain, which we called LECT, any more to
    continue the anchoring chain.

This sentence is too complicated. Let's not mention LECTs as it's not important for a changelog:
`The validators do not have to agree on the latest Exonum
    transaction anchored to Bitcoin blockchain any more to
    continue the anchoring chain.`